corehq/apps/indicators/models.py

```diff
@@ -48,8 +48,8 @@ def __init__(self, _d=None, **kwargs):
         self.class_path = self._class_path
 
     def __str__(self):
-        return "\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
-            " version: %(version)s, namespace: %(namespace)s. ID: %(indicator_id)s." % {
+        return u"\n\n%(class_name)s - Modified %(last_modified)s\n %(slug)s, domain: %(domain)s," \
```

Comment:
i think this would still need the \, it wouldn't need it only if the whole expression was wrapped in parentheses